build.sbt

```diff
@@ -102,6 +104,16 @@ lazy val `akka-management` = project
     mimaPreviousArtifactsSet
   )
 
+lazy val `akka-management-pki` = project
+  .in(file("management-pki"))
+  .enablePlugins(AutomateHeaderPlugin)
+  .settings(
+    name := "akka-management-pki",
+    libraryDependencies := Dependencies.ManagementPki,
+    // Don't enable mima until 1.1.1
```

Comment:
can we make this an issue attached to the 1.1.1 milestone?

Reply:
#896